src/components/Main/Profile.js

```diff
@@ -0,0 +1,78 @@
+import "./Main.css";
+import FollowButton from "./../FollowButton/index";
+import Info from "./Info";
+import Post from "./Post";
+
+function Profile({
+  name,
+  id,
+  profileExtention,
+  coverExtention,
+  verified = false,
+  descreption,
+  webRef,
+  followersCount,
+  followingCount,
+}) {
+  const profilePicture = require(`./../../assets/img/accounts/${id}/profile.${profileExtention}`);
+  const coverPicture = require(`./../../assets/img/accounts/${id}/cover.${coverExtention}`);
```

Comment:
why you don't use the import instead of require

Reply:
because I'm passing the id and the extensions so they're not constants and I can't use import inside of a function